tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -30,6 +30,8 @@ const syncSuccess = () => $(`${hamburgerMenuItemSelector}.sync-status .success`)
 const syncInProgress = () => $('*="Currently syncing"');
 const syncRequired = () => $(`${hamburgerMenuItemSelector}.sync-status .required`);
 const jsonError = async () => (await $('pre')).getText();
+const REPORTS_CONTENT_SELECTOR = '#reports-content';
+const reportsFastActionFAB = () => $(`${REPORTS_CONTENT_SELECTOR} .fast-action-fab-button mat-icon`);
 
 const actionBar = () => $('.detail-actions.right-pane');
 const actionBarActions = () => $$('.detail-actions.right-pane span');
```

Comment:
```suggestion
const actionBar = () => $('.detail-actions.right-pane');
const actionBarActions = () => $$('.detail-actions.right-pane span');
```

Are these still needed ? 

This file also has the `toggleActionbar` and `getActionBarLabels` methods are they needed ? 

If it's not needed the `tests/page-objects/default/contacts/contacts.wdio.page.js` calls the `commonPage.toggleActionbar`.